js/DropDownList/LoadOnDemand.md

```diff
@@ -1,15 +1,15 @@
 ---
 layout: post
-title: Load on deamand in DropDownList widget for Syncfusion Essential JS
-description: Learn here about Load on deamand in Syncfusion Essential JavaScript DropDownList widget, its elements, and more.
+title: Load on deamand in JS DropDownList | Syncfusion 
```

Comment:
Load on deamand in JS DropDownList Control | Syncfusion  - Change like this

Reply:
resolved